.github/workflows/sandbox-rebuild.yml

```
@@ -0,0 +1,28 @@
+name: SandBox Rebuild
+
+on: push
```

yamllint (1.29.0-1)

[warning] truthy value should be one of [false, true]

(truthy)